libs/watt/src/lib/configuration/watt-date-adapter.ts

```diff
@@ -21,7 +21,7 @@ const danishLocale = 'da';
 
 export class WattDateAdapter extends NativeDateAdapter {
   setActiveLocale(language: WattSupportedLocales): void {
-    this.setLocale(language === danishLocale ? 'da-DK' : 'en-GB');
+    this.setLocale(language === danishLocale ? danishLocale : 'en-GB');
```

Comment:
Question: Should en-GB then be en? or should danishLocale const be da-DK? seems od to mix them?